spec/javascripts/components/story/expanded_story/expanded_story_state_spec.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryState from 'components/story/ExpandedStory/ExpandedStoryState';
+import { states } from '../../../../../app/assets/javascripts/models/beta/story';
+
+describe('<ExpandedStoryState />', () => {
+  const onEditSpy = sinon.spy();
```

Comment:
Avoid sharing a spy like that because it will make it share state between tests since spies are stateful.